src/main/scala/nl/knaw/dans/easy/multideposit/actions/AddPropertiesToDeposit.scala

```diff
@@ -43,7 +43,8 @@ case class AddPropertiesToDeposit(row: Int, entry: (DatasetID, Dataset))(implici
 }
 object AddPropertiesToDeposit {
   // The email needs to be acquired (from LDAP) only once during the program execution
-  private var datamanagerEmailaddress: Try[String] = null
+  private var datamanagerEmailaddress: Try[String] = _
```

Comment:
To be honest with you, I had to look this up. I did the refactoring because IntelliJ told me to do so. The explanation basically can be found in [this answer](http://stackoverflow.com/a/8340917/2389405). Summary: it automatically initializes a _variable_ to it's default value.